metadata-ingestion/src/datahub/ingestion/source/sql/mysql.py

```diff
@@ -83,3 +84,14 @@ def get_platform(self):
     def create(cls, config_dict, ctx):
         config = MySQLConfig.parse_obj(config_dict)
         return cls(config, ctx)
+
+    def add_profile_metadata(self, inspector: Inspector) -> None:
+        if not self.config.profiling.enabled:
+            return
+        with inspector.engine.connect() as conn:
+            for row in conn.execute(
+                "SELECT table_schema, table_name, data_length from information_schema.tables"
```

Comment:
does this include the size of indexes associated with the table? should it?

Reply:
Interesting question. I did not add that because I am not 100% sure. But definitely we can iterate based off feedback if people think it should include that.